pride/wsize.go

```diff
@@ -1,14 +1,14 @@
 package pride
 
 import (
-  "os"
-  "golang.org/x/crypto/ssh/terminal"
+	"golang.org/x/crypto/ssh/terminal"
+	"os"
 )
 
 func windowSize() (height, width int) {
```

Comment:
We might want to swap the return order of these.

Reply:
Sounds good to me!